<launch>

  <arg name="arg1" default="true"/>
  <arg name="ptz_option" default="false"/>
  <node pkg="fixed_hk_sdk" type="fixed_hk_sdk_node" name="fixed_hk_sdk_node" respawn="false" output="screen">
      <param name="camera_id" type="int" value="1"/>
      <param name="sdkcom_dir" value="/home/li/fixed_ws/src/fixed_hk_sdk/lib"/>
      <param name="heartbeat_topic_string" value="/yd/heartbeat"/>
      <param name="device_ip" value="192.168.1.68"/>
      <param name="device_port" value="8000"/>
      <param name="device_username" value="admin"/>
      <param name="device_password" value="123qweasd"/>
      <param name="pan_range" type="str" value="0,360"/>
      <param name="tilt_range" type="str" value="-5,90"/>       
      <param name="heartbeat_topic_string" type="str" value="/yd/heartbeat"/>
      <param name="set_infrared_focus" type="bool" value="true"/>
      <param name="set_infrared_focus_waittime" type="int" value="20"/>
      <param name="ptz_topic" value="/fixed/platform/position"/>
      <param name="ptz_server_name" value="/fixed/platform/cmd"/>
      <param name="pub_raw_temp" type="bool" value="true"/>
      <param name="raw_temp_topic" value="/fixed/infrared/raw"/>
      <param name="auto_zoom" type="bool" value="$(arg arg1)"/>
      <param name="ptz_option" type="bool" value="$(arg ptz_option)"/>
      <param name="infrared_channel" value="1"/>
  </node>
</launch>


